concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -104,6 +114,8 @@ datadog_app_key: ${datadog_app_key:-}
 enable_datadog: ${ENABLE_DATADOG}
 enable_paas_dashboard: ${ENABLE_PAAS_DASHBOARD:-false}
 deploy_roadmap: ${DEPLOY_ROADMAP:-false}
+deploy_rubbernecker: ${DEPLOY_RUBBERNECKER:-false}
+pivotal_project_id: ${PIVOTAL_PROJECT_ID:-1275640}
```

Comment:
Where would that live? In credentials repo?
We have other hard coded values, mainly in Makefile. Would that be better there?